s64da_benchmark_toolkit/netdata.py

```diff
@@ -8,6 +8,21 @@
 
 LOG = logging.getLogger()
 
+def is_netdata_set_and_runnning(config):
+    netdata_config = config.get('netdata')
```

Comment:
Maybe even shorten to:

    netdata_url = config.get('netdata').get('url')
    if netdata_url:
        ...